setup.py

```diff
@@ -5,44 +5,50 @@
+    with open(path.join(curr_dir, "README.rst"), encoding="utf-8") as file:
         long_description = file.read()
-    
-    setup(name='region',
-          version=VERSION,
-          description='Package offering regionalization algorithms',
-          long_description=long_description,
-          url='https://github.com/pysal/region',
-          maintainer="PySAL Developers",
-          maintainer_email='pysal-dev@googlegroups.com',
-          license='BSD 3-clause',
-          # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
-          # 3 - Alpha; 4 - Beta; 5 - Production/Stable
-          classifiers=['Development Status :: 3 - Alpha',
-                       'Intended Audience :: Science/Research',
-                       'Intended Audience :: Developers',
-                       'Intended Audience :: Education',
-                       'Topic :: Scientific/Engineering',
-                       'Topic :: Scientific/Engineering :: GIS',
-                       'License :: OSI Approved :: BSD License',
-                       'Programming Language :: Python :: 3',
-                       'Programming Language :: Python :: 3.5',
-                       'Programming Language :: Python :: 3.6'],
-          keywords='regionalization spatial clustering',
-          packages=find_packages(exclude=['contrib', 'doc', 'test*']),
-          install_requires=['geopandas',
-                            'libpysal',
-                            'networkx<2.0.0',
-                            'numpy>=1.10.4',
-                            'pulp',
-                            'pytest',
-                            'scipy',
-                            'sklearn'],
-          python_requires='>3.4')
+
+    setup(
+        name="region",
+        version=VERSION,
+        description="Package offering regionalization algorithms",
+        long_description=long_description,
+        url="https://github.com/pysal/region",
+        maintainer="PySAL Developers",
+        maintainer_email="pysal-dev@googlegroups.com",
+        license="BSD 3-clause",
+        # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        # 3 - Alpha; 4 - Beta; 5 - Production/Stable
+        classifiers=[
+            "Development Status :: 3 - Alpha",
+            "Intended Audience :: Science/Research",
+            "Intended Audience :: Developers",
+            "Intended Audience :: Education",
+            "Topic :: Scientific/Engineering",
+            "Topic :: Scientific/Engineering :: GIS",
+            "License :: OSI Approved :: BSD License",
+            "Programming Language :: Python :: 3",
+            "Programming Language :: Python :: 3.5",
+            "Programming Language :: Python :: 3.6",
+        ],
+        keywords="regionalization spatial clustering",
+        packages=find_packages(exclude=["contrib", "doc", "test*"]),
+        install_requires=[
+            "geopandas",
+            "libpysal",
+            "networkx>=2.0.0",
```

Comment:
This is the actual only change in this commit, right?

Reply:
No. There are a bunch of changes. https://github.com/pysal/region/pull/35/files

Reply:
But the others are related to code formatting, isn't it?

Reply:
i merged #34 for clarity in the diff. some of the changes are from black/reformatting but others look like they're for compatibility with the api from the newer networkx

Reply:
Most of them are black formatting, but there are substantive changes along the lines of https://github.com/pysal/region/pull/35/files#diff-0ebddef0485a8d8e6b6db081fcb2d99dR66

Reply:
Ok, got it